src/foam/core/AbstractDatePropertyInfo.java

```diff
@@ -0,0 +1,21 @@
+package foam.core;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZoneId;
+import java.util.Date;
+
+public abstract class AbstractDatePropertyInfo
+  extends AbstractPropertyInfo
+{
+
+  public int compareValues(java.util.Date o1, java.util.Date o2) {
```

Comment:
@kgrgreer It's called from the auto generated property infos. https://github.com/foam-framework/foam2/blob/master/src/foam/java/PropertyInfo.js#L90